magnolia-voting-app/src/main/java/com/example/JcrIsPollEditableRule.java

```diff
@@ -0,0 +1,26 @@
+package com.example;
+
+import info.magnolia.ui.api.availability.AvailabilityDefinition;
+import info.magnolia.ui.availability.rule.AbstractJcrRule;
+
+import javax.jcr.Item;
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import java.util.Calendar;
+
+public class JcrIsPollEditableRule extends AbstractJcrRule<JcrIsPollEditableRuleDefinition> {
+    public JcrIsPollEditableRule(AvailabilityDefinition availabilityDefinition, JcrIsPollEditableRuleDefinition ruleDefinition) {
+        super(availabilityDefinition, ruleDefinition);
+    }
+
+    @Override
+    protected boolean isAvailableForJcrItem(Item item) throws RepositoryException {
+        return isToDoStatus((Node) item);
+    }
+
+    public boolean isToDoStatus(Node source) throws RepositoryException {
```

Comment:
As I created PollStatusValueProvider class for setting the poll status column's value, I suggest using this code snippet instead of yours:
`return PollStatusValueProvider.getPollStatus(source).equals(PollStatusValueProvider.TO_DO);`

Reply:
I have changed my code. I have created the PollUtils class which generates poll status. There code snippet I suggested now changed by this one: 
`return PollUtils.pickStatus(source).equals(PollUtils.TO_DO);`

Reply:
I will test and apply for that when you have done your PollUtils ASAP

Reply:
It's not very nice. First, you expose static method for provider class. It seems that we are mixing its responsibility and not very well organized (coupling classes). These logic can be isolate in an util class, actually using util with static modifier still have its problems (you can search pros vs cons of using static). However, it not a big deal if we use it wisely

Also, move constants to constant class or enum class. Should be done in FA-12 @nanoeagle

Reply:
I have moved the constants into an enum class. @username-david You can follow my newest code.